mungegithub/github/github.go

```diff
@@ -1562,12 +1562,11 @@ func (obj *MungeObject) GetPR() (*github.PullRequest, bool) {
 		return obj.pr, true
 	}
 	if !obj.IsPR() {
-		glog.Errorf("Issue: %d is not a PR", *obj.Issue.Number)
 		return nil, false
```

Comment:
It would be a good refactor to switch all the callers to use IsPR directly where appropriate. But this is the minimal change to make it less broken.